include/rmm/mr/device_memory_resource.hpp

```diff
@@ -22,6 +22,18 @@
 using cudaStream_t = struct CUstream_st*;
 
 namespace rmm {
+
+namespace detail {
+/**
+ * Align up to a power of 2, align_bytes is expected to be a nonzero power of 2
+ */
+inline std::size_t align_up(std::size_t v, std::size_t align_bytes) noexcept
```

Comment:
This should be documented consistent with https://github.com/rapidsai/cudf/blob/branch-0.11/cpp/src/examples/documentation_example.cpp

Reply:
Updated. Couldn't find the eyeroll emoji in in github.